src/webhint-theme/layout/scan.ejs

```diff
@@ -33,11 +34,17 @@
     <%- include('partials/scripts', { theme }); %>
 
     <% if(result) { %>
+        <a href="#main" class="anchor-top" hidden aria-label="back to top">top</a>
+
         <!-- build:js /static/scripts/highlight.js -->
         <script src="/js/highlight/highlight.min.js"></script>
         <script src="/js/highlight/languages/xml.min.js"></script>
         <!-- endbuild -->
 
+        <!-- build:js /static/scripts/anchor-top.js -->
+        <script async src="/components/anchor-top/anchor-top.js"></script>
```

Comment:
Are we minifying individually in the docs too?

Reply:
no, en docs, it is minified with `treeview.js`